GPIOtesting

```diff
@@ -18,10 +18,23 @@ from aiy.pins import PIN_D
 
 red = LED(PIN_D)
 
-while True:
-	red.on()
-	print('hello red')
-	sleep(2)
-	red.off()
-	print('goodbye red')
-	sleep(2)
+try:
```

Comment:
Unless its a language restriction, I recommend using 2 spaces for each indentation.  It gives more visible information using less horizontal space on a monitor.